### DISCUSSION TITLE
The homepage is useless

### DISCUSSION THREAD

**Discussion by gabriel-piltzer-trimble** [community/community#123802] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

As a developer the most common tasks I have when navigating directly to github.com are:
1. Quickly navigate to a repository in one of the organizations I belong to
2. Review a PR that I have been tagged as a reviewer on

Instead all that I see are release announcements and other links that I would never click on. I suggest either making the homepage customizable with the items I listed above included or add those 2 items to the homepage.

**Comment by Jv2350** [comment]
👍👍👍👍

**Comment by Atharvkote** [comment]
Yeah Partially Agree Cause you can Nagivate the any platform available, can only Access repos that it ,that kinda sucks😑
